swiftgen-cli/storyboards.swift

```diff
@@ -15,19 +15,21 @@ let storyboardsCommand = command(
     description: "The name of the enum to generate for Scenes"),
   Option<String>("segueEnumName", "StoryboardSegue", flag: "g",
     description: "The name of the enum to generate for Segues"),
-  Options<String>("import", [], count: 0,
+  VariadicOption<String>("import", [],
```

Comment:
Yeah I know that @plivesey had the same misunderstanding of this `Options<…>(count: )` stuff in `Commander` (and so did I when we talked about it)… not very intuitive ^^